internal/projects/retrieve.go

```diff
@@ -104,5 +102,7 @@ func (c *Client) Retrieve() *cobra.Command {
 
 	retrieveProjectCmd.Flags().StringVarP(&projectName, "project", "n", "", "The name of the project.")
 	retrieveProjectCmd.Flags().StringVarP(&projectID, "id", "i", "", "The project's UUID, which can be specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.")
+	retrieveProjectCmd.MarkFlagsMutuallyExclusive("id", "project")
```

Comment:
This is the only change that is not `Args: cobra.NoArgs,`.

This takes advantage of `MarkFlagsMutuallyExclusive` to replace the conflicting id/project field che k in `projects retrieve`. This is the only location where this Cobra helper seemed to fit.

Reply:
Ahh. The second one is in this commit/cherry-pick. The first one was included in #485 and would be removed from this PR in a rebase.